website/static/js/share/results.js

```diff
@@ -1,126 +1,252 @@
+                    utils.updateFilter(vm, 'match:contributors.givenName:' + contributor.givenName, true);
+                }
+            }, contributor.name)
+        ]);
+    }
+};
+
+var Subjects = {
+    view: function(ctrl, params){
+        var result = params.result;
+        var subjectViews = $.map(result.subjects || [], function(subject, i) {
+            return m.component(Subject, $.extend({subject: subject}, params));
+        });
+        if (result.showAllsubjects || (result.subjects || []).length <= 5) {
+            return m('span', subjectViews);
+        }
+        return m('span', [
+            subjectViews.slice(0, 5),
+            m('br'),
+            m('div', m('a', {onclick: function() {result.showAllsubjects = result.showAllsubjects ? false : true;}},'See All'))
+        ]);
+
+    }
+};
+
+var Subject = {
+    view: function(ctrl, params) {
+        var subject = params.subject;
+        var vm = params.vm;
+        return m('span', m('.badge.pointer', {onclick: function(){
+                utils.updateFilter(vm, 'match:subjects:"' + subject + '"', true);
+            }}, subject.length < 50 ? subject : subject.substring(0, 47) + '...'),
+            ' ');
+    }
+};
+
+var Footer = {
+    controller: function(params) {
+        var self = this;
+        this.result = params.result;
+        this.cleanResult = m.prop(null);
+        this.loadRawNorm = function() {
+            utils.loadRawNormalized(this.result).then(
+                function(cleanData) {
+                    self.cleanResult(cleanData);
+                }, function(error) {
+                    self.cleanResult(error);
+                }
+            );
+        };
+    },
+    view: function(ctrl, params) {
+        var result = params.result;
+        var vm = params.vm;
+        return m('div', [
+            m('span',
+                'Released on ' + new $osf.FormattableDate(result.providerUpdatedDateTime).local,
+                vm.rawNormedLoaded() ?  m('span', [
+                    m('span', {style: {'margin-right': '5px', 'margin-left': '5px'}}, ' | '),
+                    m('a', {
+                        onclick: function() {
+                            ctrl.showRawNormed = ctrl.showRawNormed ? false : true;
+                            ctrl.loadRawNorm();
+                        }
+                    }, 'Data')
+                ]) : m('span', [])
+            ),
+            m('span.pull-right', [
+                m('img', {src: vm.ProviderMap[result.shareProperties.source].favicon, alt: 'favicon for ' + result.shareProperties.source, style: {width: '16px', height: '16px'}}),
+                ' ',
+                m('a', {onclick: function() {utils.updateFilter(vm, 'shareProperties.source:' + result.shareProperties.source);}}, vm.ProviderMap[result.shareProperties.source].long_name),
+                m('br')
+            ]),
+            ctrl.showRawNormed ? m.component(RawNormalizedData, {result: ctrl.cleanResult}) : '',
```

Comment:
Will define ```showRawNormed``` in the controller method! I do think it's necessary, because it defines if the raw normalized pane is open or not, when clicking "Data." Clicking Data opens a separate section that allows you to see raw and normalized data in a tabbed table, and close it again. 
![data button](https://cloud.githubusercontent.com/assets/801594/9138190/4ab885ae-3cf0-11e5-8115-796705a2c479.gif)
Perhaps there's a better way that I'm missing, though, so happy to change it!


I had been calling the ```m.prop``` on the other side - once it got passed tot he RawNormalizedData component - makes more sense to call it here though!